server/src/test/resources/server_test_data.sql

```diff
@@ -1,104 +0,0 @@
---
```

Comment:
Should we have a command to start the server and fill the database with the test data? I want to avoid having the same data in two places. Jooq doesn't seem to have an API that takes a sql script as input. I can try reading the file and calling `DSLContext.html.execute(java.lang.String)` but different databases might not like the date format.